service/raft_topology.hh

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Modified by ScyllaDB
```

Comment:
This does needed in the files we add. Many files in the code base are translated from Cassandra and hence have their license with parts modified by ScyllaDB and this is what the comment says. This is not true for new stuff.

Reply:
I also have it in my patch set and just removed it :)

Reply:
ok, will remote this header